tests/test_quota.py

```diff
@@ -103,7 +102,7 @@ def test_dispatcher_role_propagated(setup_spark, role, enforce_role):
 
 @pytest.mark.sanity
 @pytest.mark.parametrize('role,enforce_role', [
-    ("spark_role", True)
```

Comment:
Custom role enforcement is a valid use case for earlier DCOS versions (pre-2.0). What if we refactor the tests and split them into two:
- _a_ role enforced (DCOS pre-2.0)
- group role enforced (DCOS 2.0 and higher)

Tests can be marked with `@pytest.mark.dcos_min_version('version')`